client/state/action-types.js

```diff
@@ -50,6 +50,7 @@ export const RECEIPT_FETCH_FAILED = 'RECEIPT_FETCH_FAILED';
 export const REMOVE_NOTICE = 'REMOVE_NOTICE';
 export const SELECTED_SITE_SET = 'SELECTED_SITE_SET';
 export const SERIALIZE = 'SERIALIZE';
+export const SERVER_DESERIALIZE = 'SERVER_DESERIALIZE';
```

Comment:
I'm not attached to the name, so I'd be happy to open up another PR if we find that another term is more clear.